src/services/fireblocks.service.js

```diff
@@ -91,14 +91,24 @@ async function getTxSignatures(tx, storage, client) {
     });
   }
 
-  /* istanbul ignore next */
-  if (tx.version === hathorLib.constants.NANO_CONTRACTS_VERSION) {
+  /** @type {hathorLib.Address} */
+  let address = null;
```

Comment:
These changes to fireblocks are due to the changes on the lib on how we sign the nano contract transactions